### PR TITLE
Add php xlswriter extension

### PR DESCRIPTION
**Pull request by Yohoed**
## Description

This PR allows to install php xlswriter extension do php-fpm,workspace and php-worker.
## Motivation and Context

Use laradock and laravel for technical and to generate excel.
## Types of Changes

- [] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## Definition of Done Checklist:

- [] I've read the [Contribution Guide](http://laradock.io/contributing).
- [] I've updated the **documentation**. (refer to [this](http://laradock.io/contributing/#update-the-documentation-site) for how to do so).
- [x] I enjoyed my time contributing and making developer's life easier :)

### CODE REVIEW

**Comment by Yohoed** [comment]
The xlswriter github Url https://github.com/viest/php-ext-xlswriter

File: php-fpm/Dockerfile [hunk 1]
```diff
@@ -488,6 +488,23 @@ RUN if [ ${INSTALL_GEARMAN} = true ]; then \
...
+    if [ ${INSTALL_XLSWRITER} = true ]; then \
+      # Install Php xlswriter Extension \
+      if [ $(php -r "echo PHP_MAJOR_VERSION;") > "5" ]; then \
+          pecl install xlswriter; \
+          docker-php-ext-enable xlswriter; \
+          php -m | grep -q 'xlswriter'; \
+      else \
+          echo "PHP Extension for xlswriter is not supported for PHP 5.0"\
+      fi \
+    ;fi
```
Comment (erikn69) [inline php-fpm/Dockerfile:506]: This breaks the tests, maybe must be:
```batch
ARG INSTALL_XLSWRITER=false
RUN set -eux; \
    if [ ${INSTALL_XLSWRITER} = true ]; then \
      # Install Php xlswriter Extension \
      if [ $(php -r "echo PHP_MAJOR_VERSION;") > "5" ]; then \
          pecl install xlswriter  && \
          docker-php-ext-enable xlswriter  && \
          php -m | grep -q 'xlswriter'; \
      else \
          echo "PHP Extension for xlswriter is not supported for PHP 5.0"; \
      fi \
    ;fi
```

File: workspace/Dockerfile [hunk 2]
```diff
@@ -565,6 +565,27 @@ RUN set -eux; \
...
+###########################################################################
+
+ARG INSTALL_XLSWRITER=false
+RUN set -eux; \
+    if [ ${INSTALL_XLSWRITER} = true ]; then \
+      # Install Php xlswriter Extension
+      if [ $(php -r "echo PHP_MAJOR_VERSION;") > "5" ]; then \
+        echo '' | pecl -q install xlswriter; \
+        echo "extension=xlswriter.so" >> /etc/php/${LARADOCK_PHP_VERSION}/mods-available/xlswriter.ini; \
+        ln -s /etc/php/${LARADOCK_PHP_VERSION}/mods-available/xlswriter.ini /etc/php/${LARADOCK_PHP_VERSION}/cli/conf.d/20-xlswriter.ini; \
```
Comment (erikn69) [inline workspace/Dockerfile:581]:
``` batch
echo '' | pecl -q install xlswriter && \
echo "extension=xlswriter.so" >> /etc/php/${LARADOCK_PHP_VERSION}/mods-available/xlswriter.ini && \
```

File: php-worker/Dockerfile [hunk 3]
```diff
@@ -367,6 +367,24 @@ RUN set -eux; \
...
+###########################################################################
+
+ARG INSTALL_XLSWRITER=false
+
+RUN set -eux; \
+    if [ ${INSTALL_XLSWRITER} = true ]; then \
+      # Install Php xlswriter Extension \
+      if   [ $(php -r "echo PHP_MAJOR_VERSION;") > "5" ]; then \
+          pecl install xlswriter; \
+          docker-php-ext-enable xlswriter; \
```
Comment (erikn69) [inline php-worker/Dockerfile:381]: Same as others
```batch
pecl install xlswriter && \
docker-php-ext-enable xlswriter && \
```

**Comment by erikn69** [comment]
Also add value for `.env` on testing, example
```batch
sed -i -- 's/XLSWRITER=true/XLSWRITER=false/g' .env 
```
https://github.com/laradock/laradock/blob/a70675007dd5da7097f46f118a84598c056d7311/.github/workflows/main-ci.yml#L29-L37

**Pull request merged**